docs/Tuners.md

```diff
@@ -0,0 +1,324 @@
+
+> Builtin Tuner Name: **Grid Search**
+
+**Suggested scenario**
+
+Note that the only acceptable types of search space are `choice`, `quniform`, `qloguniform`. **The number `q` in `quniform` and `qloguniform` has special meaning (different from the spec in [search space spec](./SearchSpaceSpec.md)). It means the number of values that will be sampled evenly from the range `low` and `high`.**
+
+It is suggested when search space is small, it is feasible to exhaustively sweeping the whole search space.
+
+**Usage example**
+
+```yaml
+# config.yml
+tuner:
+  builtinTunerName: GridSearch
+```
+
+<br>
+
+<a name="Hyperband"></a>
+
+![#1589F0](https://placehold.it/15/1589F0/000000?text=+) `Hyperband`
+
+> Builtin Advisor Name: **Hyperband**
+
+**Suggested scenario**
+
+It is suggested when you have limited computation resource but have relatively large search space. It performs good in the scenario that intermediate result (e.g., accuracy) can reflect good or bad of final result (e.g., accuracy) to some extent.
+
+**Requirement of classArg**
+
+* **optimize_mode** (*maximize or minimize, optional, default = maximize*) - If 'maximize', tuners will return the hyperparameter set with larger expectation. If 'minimize', tuner will return the hyperparameter set with smaller expectation.
+* **R** (*int, optional, default = 60*) - the maximum STEPS (could be the number of mini-batches or epochs) can be allocated to a trial. Each trial should use STEPS to control how long it runs.
+* **eta** (*int, optional, default = 3*) - proportion of discarded trials
+
+**Usage example**
+
+```yaml
+# config.yml
+advisor:
+  builtinAdvisorName: Hyperband
+  classArgs:
+    optimize_mode: maximize
+    R: 60
+    eta: 3
+```
+
+<br>
+
+<a name="NetworkMorphism"></a>
+
+![#1589F0](https://placehold.it/15/1589F0/000000?text=+) `Network Morphism`
+
+> Builtin Tuner Name: **NetworkMorphism**
+
+**Installation**
+
+NetworkMorphism requires [pyTorch](https://pytorch.org/get-started/locally), so users should install it first.
+
+**Suggested scenario**
+
+It is suggested that you want to apply deep learning methods to your task (your own dataset) but you have no idea of how to choose or design a network. You modify the [example](../examples/trials/network_morphism/cifar10/cifar10_keras.py) to fit your own dataset and your own data augmentation method. Also you can change the batch size, learning rate or optimizer. It is feasible for different tasks to find a good network architecture. Now this tuner only supports the cv domain.
+
+**Requirement of classArg**
+
+* **optimize_mode** (*maximize or minimize, optional, default = maximize*) - If 'maximize', tuners will return the hyperparameter set with larger expectation. If 'minimize', tuner will return the hyperparameter set with smaller expectation.
+* **task** (*('cv'), optional, default = 'cv'*) - The domain of experiment, for now, this tuner only supports cv domain.
```

Comment:
it's better to expand cv in description.

Reply:
Done